package com.syntax.class34;

public class TestException {

	public static void main(String[] args) {
		TryCatchFinally.division(10, 2);
		TryCatchFinally.division(22, 0);
System.out.println("----End of the code---- ");
		TryCatchFinally.read("");
	}

}
